debian/control

```diff
@@ -339,3 +339,16 @@ Description: architecture for analyzing software, report importer
  type identification.
  .
  This package contains the reportImport agent programs and their resources.
+
+Package: fossology-softwareHeritage
+Architechture: any
+Depends: fossology-ununpack, fossology-wgetagent, ${misc:Depends}
+Description: architecture for fetching the origin details of the files
+ of a package from software heritage archive.
```

Comment:
Also, the package description should be in a single line, followed by long description in multiple lines.